### PR TITLE
Use react-helmet-async to update meta description and canonical link

### PR DESCRIPTION
**Pull request by zhx828**
This is related to https://github.com/oncokb/oncokb-pipeline/issues/321

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -83,6 +82,8 @@
   },
   "devDependencies": {
     "@babel/core": "7.6.4",
+    "@babel/plugin-proposal-class-properties": "^7.18.6",
+    "@babel/plugin-proposal-optional-chaining": "^7.21.0",
```
Comment (jfkonecn) [inline package.json:86]: These packages appear to be deprecated <https://www.npmjs.com/package/@babel/plugin-proposal-class-properties>, <https://www.npmjs.com/package/@babel/plugin-proposal-optional-chaining>

Comment (jfkonecn) [reply package.json:86]: Why are these needed now? How did we get this far without them?

Comment (zhx828) [reply package.json:86]: These two are required by the react-helmet

File: src/main/webapp/app/pages/alterationPage/AlterationPage.tsx [hunk 2]
```diff
@@ -168,7 +169,20 @@ export default class AlterationPage extends React.Component<
 
   render() {
     return (
-      <DocumentTitle title={this.documentTitle}>
+      <>
+        <Helmet>
+          <title>{this.documentTitle}</title>
+          <link
```
[the remark this answers is not in the document]
Comment (zhx828) [reply src/main/webapp/app/pages/alterationPage/AlterationPage.tsx:175]: We have description in AnnotationPage which is a subcomponent under AlterationPage. But good point, I will verify if a page does not have description, whether the one from previous page will be used.

[the remark this answers is not in the document]
Comment (zhx828) [reply src/main/webapp/app/pages/alterationPage/AlterationPage.tsx:175]: Interestingly react-helmet-async will help cleaning up the meta tag if the new page does not have description.

[the remark this answers is not in the document]
Comment (jfkonecn) [reply src/main/webapp/app/pages/alterationPage/AlterationPage.tsx:175]: That's nice. It'll make the code way easier to maintain.

**Review by jfkonecn — APPROVED** [review]

**Pull request merged**